dist/s_longlines

```diff
@@ -9,6 +9,7 @@ check_fast_mode_flag
 l=`(find bench/wtperf examples ext src test tools/checksum_bitflip -name '*.[chsy]' &&
     find dist -name '*.py' &&
     find src -name '*.in') |
+    git check-ignore --stdin -nv | grep -E "^::" | cut -f2 |
```

Comment:
It's not obvious to me what this line of change is trying to achieve. Can you help to elaborate?

Reply:
Got it, thanks